reference/constraints/AtLeastOneOf.rst

```diff
@@ -60,6 +60,34 @@ The following constraints ensure that:
             protected $grades;
         }
 
+    .. code-block:: php-attributes
+
+        // src/Entity/Student.php
+        namespace App\Entity;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Student
+        {
+            #[
+                Assert\AtLeastOneOf([
+                    new Assert\Regex('/#/'),
+                    new Assert\Length(min: 10),
+                ])
+            ]
+            protected $password;
```

Comment:
We should rename this to `plainPassword`, in nearly every application, this should be a hash, which does not need to be validated 🤔

Reply:
You're right, making the change!